src/blog/2023/03/ibmcloud-starter-removed.md

```diff
@@ -0,0 +1,70 @@
+
+## Integrations
+
+If you've seen the excellent ["Create a Node-RED starter application"](https://developer.ibm.com/tutorials/how-to-create-a-node-red-starter-application/) article on IBM Developer, you'll probably be looking to connect up to a Cloudant Instance, or other IBM Cloud Services. Don't worry. All of that is still available through Node-RED on FlowForge.
+
+You can install the relevant nodes in one of two places:
+
+1. **Node-RED Palette Manager:** Click "Menu > Manage Palette > Install". The menu is available via the icon in the top-right of your running Node-RED Instance)
+
+![Screenshot of Node-RED's Manage Palette menu](./images/nr-manage-palette-cloudant.png "Screenshot of Node-RED's Manage Palette menu")
+
+
+2. **FlowForge Instance Settings:** For a given Instance in FlowForge, click "Settings > Palette". You can then define the npm module name and versions explicitely in the "Installed Modules" section
+
+![Screenshot of FlowForge's "Installed Modules" option in Instance > Settings > Palette](./images/ff-installed-modules.png "Screenshot of FlowForge's 'Installed Modules' option in Instance > Settings > Palette")
+
+It's also easy to setup [Environment Variables](/blog/2023/01/environment-variables-in-node-red/) for Node-RED in FlowForge for when you integrate with external services like APIs too.
+
+## Security
+
+As with IBM Cloud, FlowForge makes it very easy to secure your Node-RED Applications. FlowForge offers three tiers of security options on your Node-RED Instances to secure any exposed HTTP routes on your Node-RED instance, e.g. REST API endpoints or Node-RED Dashboard.
+
+- **None**: Anyone will be able to access the exposed routes.
+- **Basic Auth**: Setup a single, dedicated username and password combination that is required in order to access the routes.
+- **FlowForge Credentials**: Visitors can use their FlowForge username/password in order to access the endpoints. This also includes SSO if you have that configured for your FlowFoerge Team.
+
+## Migrating Existing Instances
+
+If you're looking to move your Node-RED applications from IBM Cloud, then you can do so through one of two options:
+
+### Node-RED Tools Plugin
+
+You can use our Node-RED Tools plugin to migrate your flows and credentials over to FlowForge. You can read the details in our [Migration Guide](https://flowforge.com/docs/user/migration/), which also includes instructions on how to export your environment variables too.
+
+
+### Manual Import
+
+This will only enable you to import your Flows, not the associated credentials.
+
+1. Export your existing `flows.json` from your IBM Cloud-hosted Node-RED instance by choosing "Export > All Flows > Download" within Node-RED
+2. Create a new Application on FlowForge
+3. Once created, click the Node-RED instance that has been generated within your Application
+4. Click "Settings"
+5. Scroll down and select the "Import Instance" option
+6. Choose your `flows.json` file that you downloaded earlier.
```

Comment:
You're missing the environment variables? Or doesn't IBM Cloud use those?

Reply:
If you're doing this manually, then no easy way of exporting from IBM Cloud that we are aware of (without being able to go in and experiment)